### ISSUE TITLE
"Invalid cast" error while installing SteamVR Driver

### ISSUE THREAD

**Issue by ruddnev** [Timocop/PSMoveServiceEx-Virtual-Device-Manager#4]
While installing SteamVR Driver on a fresh Windows 11 system, on pressing "Register driver" a window pops up with "Invalid cast" error. I fixed the error by installing iVRy SteamVR driver and running it once.
_I think the problem is with present but empty "external_drivers"._ 

Thank you for your continuation of PSMoveService!

**Comment by Timocop** [comment]
This has been fixed in the latest release. Please update to the newest version.

**Comment by ruddnev** [comment]
I am on 8.3.1.0, so the latest one.

**Comment by Timocop** [comment]
Could you upload your ``AppData\Local\openvr\openvrpaths.vrpath`` file please?

**Comment by ruddnev** [comment]
There you go.

```
{
	"config":	[
		"C:\\Program Files (x86)\\Steam\\config"
	],
	"external_drivers":	[
		"C:\\Program Files (x86)\\Steam\\steamapps\\common\\iVRy",
		"C:\\4nswers\\VR\\PSMSVirtualDeviceManager\\driver\\psmsex"
	],
	"jsonid":	"vrpathreg",
	"log":	[
		"C:\\Program Files (x86)\\Steam\\logs"
	],
	"runtime":	[
		"C:\\Program Files (x86)\\Steam\\steamapps\\common\\SteamVR"
	],
	"version":	1
}
```

**Comment by Timocop** [comment]
This issue with "Invalid cast" should not happen anymore since v8.3.0.0. I have tested it with tons of different scenarios.
But if you encounter this issue again, feel free to reopen this issue again and post the file again.
Solved, closed.

**Comment by Timocop** [comment]
Ok i have found the issue. Seems like this issue only happens when ``external_drivers`` is ``null`` for whatever reason.
Probably caused (in my case) by ALVR v21 nuking all external drivers for some reason.

```
{
	"config" : 
	[
		"C:\\Program Files (x86)\\Steam\\config"
	],
	"external_drivers" : null,
	"jsonid" : "vrpathreg",
	"log" : 
	[
		"C:\\Program Files (x86)\\Steam\\logs"
	],
	"runtime" : 
	[
		"C:\\Program Files (x86)\\Steam\\steamapps\\common\\SteamVR"
	],
	"version" : 1
}
```

**Comment by ruddnev** [comment]
That's the thing, I did not install ALVR. 
So, I had a fresh Windows 11 install, I had SteamVR installed but didn't run it once. I tried to register the driver, then I encountered the problem. I ran SteamVR, and then tried again to no avail. 
And only when installing and running iVRy SteamVR driver (I didn't need to install it, I just dug through the code and thought it could fill the external drivers) I could register the driver. 
I don't have any antiviruses, and I don't know what else could interfer, and that's why I think that people can come across this issue when running on a clean setup, just like me
